test/sequential/test-repl-persistent-history.js

```diff
@@ -170,7 +170,7 @@ function runTest() {
         if (output.charCodeAt(0) === 27 || /^[\r\n]+$/.test(output))
           return next();
 
-        assert.strictEqual(output, expected.shift());
+        assert.deepStrictEqual(output, expected.shift());
```

Comment:
Ah, I could've sworn that was passing on master...it isn't though. This change will be removed.